pepys_import/file/file_processor.py

```diff
@@ -45,6 +45,7 @@ def __init__(self, filename=None):
             if not os.path.exists(ARCHIVE_PATH):
```

Comment:
Now that we're using the config file, we can remove support for the ARCHIVE_PATH environment variable, please (including tests).  This will simplify the maintenance/configuration of Pepys.

Reply:
Actually, this is a variable that comes from the config file (`config.py`). So, it is not an environment variable.

Reply:
I think you were confused because of the comment. I'm going to change it now

Reply:
Done:
https://github.com/debrief/pepys-import/pull/216/commits/246be9b009630fba3bccc358177d3c7480be4010